components/OnTodaysShow.vue

```diff
@@ -90,8 +100,38 @@ export default {
           url: 'http://www.google.com'
         }
       ],
-      segmentsToShow: 6
+      segmentsToShow: 6,
+      social: [
+        {
+          contact: 'AllOfItWNYC',
+          service: 'facebook'
+        },
+        {
+          contact: 'AllOfItWNYC',
+          service: 'twitter'
+        },
+        {
+          contact: 'allofitwnyc',
+          service: 'instagram'
+        }]
     }
   }
 }
 </script>
+<style scoped>
```

Comment:
I don't care about putting styles here, but I have a feeling Matt might want them all in the design system... lol